e2e/cypress/integration/enterprise/system_console/compliance/compliance_export_ui_spec.js

```diff
@@ -0,0 +1,228 @@
+        cy.findByTestId('FileSettings.AmazonS3AccessKeyIdinput').clear().type(minioAccessKey);
+        cy.findByTestId('FileSettings.AmazonS3SecretAccessKeyinput').clear().type(minioSecretKey);
+        cy.findByTestId('FileSettings.AmazonS3Endpointinput').clear().type(minioS3Endpoint);
+        cy.findByTestId(`FileSettings.AmazonS3SSL${minioS3SSL}`).check();
+
+        // # Save file storage settings
+        cy.findByTestId('saveSetting').click();
+        waitUntilConfigSave();
+
+        // # Test connection and verify that it's successful
+        cy.findByRole('button', {name: 'Test Connection'}).click();
+        cy.findByText('Connection was successful').should('be.visible');
+
+        // # Go to compliance page and enable export
+        cy.uiGoToCompliancePage();
+        cy.uiEnableComplianceExport();
+        cy.uiExportCompliance();
+
+        // # Navigate to a team and post an attachment
+        gotoTeamAndPostImage();
+
+        // # Go to compliance page and start export
+        cy.uiGoToCompliancePage();
+        cy.uiExportCompliance();
+
+        // # Get the first row
+        cy.get('.job-table__table').find('tbody > tr').eq(0).as('firstRow');
+
+        // # Get the download link
+        cy.get('@firstRow').findByText('Download').parents('a').should('exist').then((fileAttachment) => {
+            const fileURL = fileAttachment.attr('href');
+
+            // * Download link should not exist this time
+            cy.apiDownloadFileAndVerifyContentType(fileURL);
+        });
+    });
+
+    it('MM-T1168 - Compliance Export - Run Now, entry appears in job table', () => {
+        // # Go to compliance page and enable export
+        cy.uiGoToCompliancePage();
+        cy.uiEnableComplianceExport();
+        cy.uiExportCompliance();
+
+        // # Navigate to a team and post an attachment
+        gotoTeamAndPostImage();
+
+        // # Go to compliance page and start export
+        cy.uiGoToCompliancePage();
+        cy.uiExportCompliance();
+
+        // # Get the first row
+        cy.get('.job-table__table').find('tbody > tr').eq(0).as('firstRow');
+
+        // * Verify table header
+        cy.get('@firstheader').within(() => {
+            cy.get('th:eq(1)').should('have.text', 'Status');
+            cy.get('th:eq(2)').should('have.text', 'Files');
+            cy.get('th:eq(3)').should('have.text', 'Finish Time');
+            cy.get('th:eq(4)').should('have.text', 'Run Time');
+            cy.get('th:eq(5)').should('have.text', 'Details');
+        });
+
+        // * Verify first row (last run job) data
+        cy.get('@firstRow').within(() => {
+            cy.get('td:eq(1)').should('have.text', 'Success');
+            cy.get('td:eq(2)').should('have.text', 'Download');
+            cy.get('td:eq(4)').contains('seconds');
+            cy.get('td:eq(5)').should('have.text', '1 messages exported.');
+        });
+    });
+
+    it('MM-T1169_1 - Compliance Export - CSV and Global Relay', () => {
```

Comment:
I suggests to extract the other step (step 2) into its own new test case in Zephyr. Then this can be as suggested below.

```suggestion
    it('MM-T1169 - Compliance Export - CSV and Global Relay', () => {
```